package.json

```diff
@@ -112,6 +112,7 @@
     "worker-pouch": "git+https://github.com/jkleinsc/worker-pouch.git"
   },
   "dependencies": {
+    "broccoli-funnel": "^2.0.1",
```

Comment:
Was it required?

Reply:
That was another mistake!
I imagine that occured too with the tests with semver!
As the other situation, i will rollback that!